projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -800,7 +801,14 @@ private static String getContainerName(V1Job job) {
                   .map(v1ContainerStateTerminated -> StringUtils.trim(v1ContainerStateTerminated.getMessage()))
                   .ifPresent(s -> jobExecutionStatusBuilder.terminationMessage(s));
             jobExecutionStatusBuilder.terminationReason(jobStatusCondition.getReason());
+
+            // Termination Reason of the data job pod container
+            Optional<V1ContainerStateTerminated> lastContainerState = getTerminationStatus(job);
```

Comment:
You already have the status on line 797 - that was the idea behind moving it in the `if`.

Reply:
Good catch, removed. Thanks